bukuserver/views.py

```diff
@@ -180,7 +180,7 @@ def get_detail_value(self, context, model, name):
     edit_template = "bukuserver/bookmark_edit.html"
     named_filter_urls = True
     extra_css = ['/static/bukuserver/css/' + it for it in ('bookmark.css', 'modal.css')]
-    extra_js = ['/static/bukuserver/js/' + it for it in ('page_size.js', 'last_page.js')]
+    extra_js = ['/static/bukuserver/js/' + it for it in ('page_size.js', 'last_page.js', 'filters_fix.js')]
```

Comment:
More specifically, there's a chance of this (or the next) change causing a merge conflict since the other pull-request modifies `extra_css`.